src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -217,12 +217,7 @@ def __call__(
         latents = latents.to(self.device)
 
         # set timesteps
-        accepts_offset = "offset" in set(inspect.signature(self.scheduler.set_timesteps).parameters.keys())
-        extra_set_kwargs = {}
-        if accepts_offset:
-            extra_set_kwargs["offset"] = 1
-
-        self.scheduler.set_timesteps(num_inference_steps, **extra_set_kwargs)
+        self.scheduler.set_timesteps(num_inference_steps)
```

Comment:
Should we try to override the configuration here, or are you going to update the configuration in the relevant hf repositories?